tests/connect/standalone-x509-error-0001.phpt

```diff
@@ -29,12 +28,13 @@ $opts = array(
         "allow_self_signed" => false,
         "cafile" => $SSL_DIR . "/ca.pem", /* Defaults to openssl.cafile */
         "capath" => $SSL_DIR, /* Defaults to openssl.capath */
-        "local_cert" => $SSL_DIR . "/src/libmongoc/tests/certificates/client.pem",
+        "local_cert" => $SSL_DIR . "/client.pem",
```

Comment:
Its not not correct, its just not bundled or generated anymore :)
Actually, it is incorrect indeed after we added the $SSL_DIR var.

certificates/client.pem is generated by the mongoc test suite so was probably there when the test was originally created :)